docs/developer/guidelines/pr.md

```diff
@@ -1,5 +1,10 @@
 # Pull requests
 
+!!! warning
+    The formatting notes do NOT apply if you are contributing to [integrations-core](https://github.com/DataDog/integrations-core).
+    Simply use the [`release changelog new`](../ddev/cli.md#ddev-release-changelog-new) command.
+    Add the `changelog/no-changelog` label if you don't need a changelog.
```

Comment:
I'm probably dumb but I didn't get it right on the first read. I would have said something like `In that case, use the [`release changelog new`](../ddev/cli.md#ddev-release-changelog-new) command....` to be completely sure that's what I'm supposed to do in integrations-core

Reply:
Aha, good point, I'll update the wording.